src/Components/NavBar/NavBar.tsx

```diff
@@ -83,20 +83,29 @@ export const NavBar: React.FC = track(
         */}
         <NavSection display={["none", "none", "flex"]}>
           <NavSection>
-            <NavItem href="/collect">Artworks</NavItem>
-            <NavItem href="/auctions">Auctions</NavItem>
-            <NavItem href="/galleries">Galleries</NavItem>
+            <NavItem href="/collect" doHoverTracking>
```

Comment:
Why add the `doHoverTracking` prop if all nav items are tracked on hover?

Reply:
@damassi 
Yes. Because there are somethings also using `<NavItem>`. Such as, "works for you" and some small view. For "work for you", it already has a hover tracking. I think only adding `doHoverTracking` props to those items will minimus the changing of the code. What do you think?

Reply:
I would suggest creating a new prop `onHover` that can be passed into `NavItem`, and for  that single nav item you mention that already tracks we can place the hover tracking in that callback function. Inside of NavItem, if that `onHover` prop has been passed, guard against the default hover tracking (default = what happens `doHoverTracking` is passed, at which point we can remove the `doHoverTracking` prop completely).

Basically, if most of  the items now require hover tracking, then a prop shouldn't have  to be passed because one nav item already has it. Things can be inverted. Revise the item that already has hover tracking and make it more suitable for the many. 

Does this make sense? Down to pair if needed!

Reply:
@damassi 
Hi, Chris. I made some updates. Take a look when you have time.